spras/interactome.py

```diff
@@ -0,0 +1,123 @@
+"""
+Author: Neha Talluri
+07/19/23
+
+Methods for converting from the universal input and recreating the universal output
+"""
+
+import pandas as pd
+
+
+def convert_undirected_to_directed(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    turns a graph into a fully directed graph
+    - turns every unidirected edges into a bi-directed edge
+    - with bi-directed edges, we are not loosing too much information because the relationship of the undirected egde is still preserved
```

Comment:
```suggestion
    - turns every undirected edge into a pair of directed edges
    - with the pair of directed edges edges, we are not loosing too much information because the relationship of the undirected edge is still preserved
```